xchange-huobi/src/main/java/org/knowm/xchange/huobi/dto/marketdata/HuobiCurrency.java

```diff
@@ -0,0 +1,255 @@
+    private String withdrawQuotaPerDay;
+    private String withdrawQuotaPerYear;
+    private String withdrawQuotaTotal;
+    private String withdrawStatus;
+
+    public HuobiCurrency(@JsonProperty("chain") String chain,
+                         @JsonProperty("displayName") String displayName,
+                         @JsonProperty("baseChain") String baseChain,
+                         @JsonProperty("baseChainProtocol") String baseChainProtocol,
+                         @JsonProperty("isDynamic") boolean isDynamic,
+                         @JsonProperty("depositStatus") String depositStatus,
+                         @JsonProperty("maxTransactFeeWithdraw") String maxTransactFeeWithdraw,
+                         @JsonProperty("maxWithdrawAmt") String maxWithdrawAmt,
+                         @JsonProperty("minDepositAmt") String minDepositAmt,
+                         @JsonProperty("minTransactFeeWithdraw") String minTransactFeeWithdraw,
+                         @JsonProperty("minWithdrawAmt") String minWithdrawAmt,
+                         @JsonProperty("numOfConfirmations") int numOfConfirmations,
+                         @JsonProperty("numOfFastConfirmations") int numOfFastConfirmations,
+                         @JsonProperty("transactFeeWithdraw") String transactFeeWithdraw,
+                         @JsonProperty("withdrawFeeType") String withdrawFeeType,
+                         @JsonProperty("withdrawPrecision") int withdrawPrecision,
+                         @JsonProperty("withdrawQuotaPerDay") String withdrawQuotaPerDay,
+                         @JsonProperty("withdrawQuotaPerYear") String withdrawQuotaPerYear,
+                         @JsonProperty("withdrawQuotaTotal") String withdrawQuotaTotal,
+                         @JsonProperty("withdrawStatus") String withdrawStatus) {
+        this.chain = chain;
+        this.displayName = displayName;
+        this.baseChain = baseChain;
+        this.baseChainProtocol = baseChainProtocol;
+        this.isDynamic = isDynamic;
+        this.depositStatus = depositStatus;
+        this.maxTransactFeeWithdraw = maxTransactFeeWithdraw;
+        this.maxWithdrawAmt = maxWithdrawAmt;
+        this.minDepositAmt = minDepositAmt;
+        this.minTransactFeeWithdraw = minTransactFeeWithdraw;
+        this.minWithdrawAmt = minWithdrawAmt;
+        this.numOfConfirmations = numOfConfirmations;
+        this.numOfFastConfirmations = numOfFastConfirmations;
+        this.transactFeeWithdraw = transactFeeWithdraw;
+        this.withdrawFeeType = withdrawFeeType;
+        this.withdrawPrecision = withdrawPrecision;
+        this.withdrawQuotaPerDay = withdrawQuotaPerDay;
+        this.withdrawQuotaPerYear = withdrawQuotaPerYear;
+        this.withdrawQuotaTotal = withdrawQuotaTotal;
+        this.withdrawStatus = withdrawStatus;
+    }
+
+    public String getChain() {
+        return chain;
+    }
+
+    public void setChain(String chain) {
+        this.chain = chain;
+    }
```

Comment:
DTOs should be value objects with no setters

Reply:
removed all setters